css/index.css

```diff
@@ -67,10 +165,47 @@ p {
 }
 
 .container {
-    width: 800px;
+    max-width: 1000px;
```

Comment:
Make sure you set your max-width to 800px!

Reply:
Got it covered :+1: